images/macos/software-report/SoftwareReport.Generator.ps1

```diff
@@ -97,6 +98,7 @@ if ($os.IsLessThanBigSur) {
 
 $markdown += New-MDList -Style Unordered -Lines @(
     "Pip ${pip3Version}",
+    $pipxVersion,

```

Comment:
Software report for High Sierra will be contain `Pipx` line, you should add condition for High Sierra.